client/app.js

```diff
@@ -188,45 +192,67 @@ explorer.application.module.factory('GvizDataView',
     explorer.components.widget.data_viz.gviz.getGvizDataView);
 
 
-/** Register all directives. **/
-explorer.application.module.directive('chartConfig',
-    explorer.components.widget.data_viz.gviz.ChartConfigDirective);
-explorer.application.module.directive('alertLog',
-    explorer.components.alert.AlertLogDirective);
-explorer.application.module.directive('gvizChartWidget',
-    explorer.components.widget.data_viz.gviz.gvizChart);
-explorer.application.module.directive('explorerConfig',
-    explorer.components.config.ConfigDirective);
-explorer.application.module.directive('container',
-    explorer.components.container.ContainerDirective);
-explorer.application.module.directive('containerConfig',
-    explorer.components.container.ContainerConfigDirective);
-explorer.application.module.directive('dashboard',
-    explorer.components.dashboard.DashboardDirective);
+/**
+ * Register all directives.
+ * First, general-use directives.
+ */
```

Comment:
Do these need to be registered in a particular order? If not, I think it would be clearer to order alphabetically within a category.